From 1e3b9c51611ad585ee102a75d2e14263ec113b0b Mon Sep 17 00:00:00 2001
From: Christophe Roullier <christophe.roullier@st.com>
Date: Mon, 20 Feb 2023 12:07:45 +0100
Subject: [PATCH 419/529] ARM: defconfig: add CAN configs in addons fragment

Add CAN1/CAN3 peripherals config flags in addons fragment.

Signed-off-by: Christophe Roullier <christophe.roullier@st.com>
Change-Id: I4d0d6a82db6cc5283a397a2aba97f6da75ad3b4e
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/291732
Reviewed-by: Alexandre TORGUE <alexandre.torgue@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Domain-Review: Alexandre TORGUE <alexandre.torgue@foss.st.com>
---
 arch/arm/configs/fragment-02-multiv7_addons.config | 2 ++
 1 file changed, 2 insertions(+)

--- a/arch/arm/configs/fragment-02-multiv7_addons.config
+++ b/arch/arm/configs/fragment-02-multiv7_addons.config
@@ -5,6 +5,8 @@ CONFIG_ARM_SCMI_PROTOCOL=y
 CONFIG_ARM_SMC_MBOX=y
 CONFIG_ARM_STM32_CPUIDLE=y
 CONFIG_BACKLIGHT_GPIO=y
+CONFIG_CAN_M_CAN=y
+CONFIG_CAN_M_CAN_PLATFORM=y
 CONFIG_CHR_DEV_SG=y
 CONFIG_CLKSRC_STM32_LP=y
 CONFIG_COMMON_CLK_SCMI=y
